### PR TITLE
Add organisers section as well as founders section to about page

### PR DESCRIPTION
**Pull request by yochannah**
Adds an organisers section to the main page - I didn't realise Emmy wasn't on there! 😱  and adds Paz too. 

Marking as WIP - @pazbc, I think we need your bio for this, same as what we've been asking OLS-6 participants for. 

preview shows Emmy, Malvika, Berenice, Yo, and Paz, but Paz's profile is blank

### CODE REVIEW

File: _data/people.yaml [hunk 1]
```diff
@@ -4497,6 +4497,16 @@ paulvill:
...
+pazb:
+  bio: Add Bio
+  country: Argentina
+  expertise:
+  - Social Science
+  first-name: Paz
+  last-name: Bernaldo
+  pronouns: she/her
+  twitter: pazb
+  website: Add website
```
Comment (malvikasharan) [inline _data/people.yaml:4509]:
```suggestion
pazb:
  bio: Add Bio
  country: Argentina
  expertise:
  - Social Science
  - Qualitative Research
  first-name: Paz
  last-name: Bernaldo
  pronouns: she/her
  twitter: pazb
  website: Add website
```

File: _data/people.yaml [hunk 2]
```diff
@@ -4497,6 +4497,17 @@ paulvill:
...
+  bio: Add Bio
+  country: Argentina
+  expertise:
+  - Social Science
+  - Qualitative Research
+  first-name: Paz
+  last-name: Bernaldo
+  pronouns: she/her
+  twitter: pazb
+  website: Add website
```
Comment (pazbc) [inline _data/people.yaml:4510]:
```suggestion
pazb:
  bio: Add Bio bla bla
  country: Argentina
  expertise:
  - Social Science
  - Qualitative Research
  first-name: Paz
  last-name: Bernaldo
  pronouns: she/her
  twitter: pazbyc
  website: Add website
```

File: _data/people.yaml [hunk 3]
```diff
@@ -4497,6 +4497,17 @@ paulvill:
   pronouns: he/him
   twitter: paulvilloutreix
   website: https://paulcinq.wordpress.com
+pazb:
```
Comment (pazbc) [inline _data/people.yaml:4500]:
```suggestion
pazbc:
```

**Comment by pazbc** [comment]
Hi Yo, Malvika and I added my bio :)

**Comment by yochannah** [comment]
looks great - thanks both! 🙌

**Pull request merged**